.github/workflows/pull-request.yml

```
@@ -295,7 +295,50 @@ jobs:
         with:
           path: './coverage/lcov.info'
           min_coverage: 95.0
-          
+  
+  Docker-Check:
+    needs: Test-Application
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Check if Talawa API starts in Docker
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2181:style:6:7: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)